libs/blocks/modal/decorators/langnav.js

```diff
@@ -0,0 +1,37 @@
+import { getConfig } from '../../../utils/utils.js';
+
+function handleEvent(prefix, link, config) {
+  document.cookie = `international=${prefix};path=/`;
+  sessionStorage.setItem('international', prefix);
+  fetch(link.href, { method: 'HEAD' }).then((resp) => {
+    if (!resp.ok) throw new Error('request failed');
+    window.location.assign(link.href);
+  }).catch(() => {
+    const prefixUrl = prefix ? `/${prefix}` : '';
+    window.location.assign(`${prefixUrl}${config.contentRoot || ''}/`);
```

Comment:
This won't work with the changes that @fullcolorcoder is doing in his PR:
https://github.com/adobecom/milo/pull/431
since `config.contentRoot` will contain the `prefix`.  However, it will only have the prefix when contentRoot is defined as an array.   I'll let you 2 figure out the best approach.

Reply:
thanks, I'll take look!